VRDR.Messaging/BaseMessage.cs

```diff
@@ -214,18 +236,22 @@ public string MessageDestination
         }
 
         /// <summary>Jurisdiction-assigned death certificate number</summary>
-        public string CertificateNumber
+        public uint? CertificateNumber
```

Comment:
I went back and forth on this. In the end I though an unsigned int was more appropriate because we use the same (actually a non-negative int) for void block count and thought the two should match.

Reply:
Makes sense to me